utils/debug.c

```diff
@@ -147,6 +171,25 @@ void setup_color(enum color_setting color, char *pager)
 	}
 }
 
+static const char *get_color(char code)
+{
+	unsigned i;
+
+	if (out_color != COLOR_ON)
+		return TERM_COLOR_NORMAL;
+
+	for (i = 0; i < ARRAY_SIZE(colors); i++) {
+		if (code == colors[i].code) {
+			if (format_mode == FORMAT_HTML)
+				return colors[i].html_color;
+			else
+				return colors[i].color;
+		}
+	}
+	pr_warn("get_color('%c'): invalid color code\n", code);
```

Comment:
I'm not sure we want to add a warning here... it'd generate too many warnings if something is broken.

Reply:
Yeap, removed it.